### PR TITLE
Static pages

### PR DESCRIPTION
**Pull request by FuhuXia**
Part of
- https://github.com/GSA/data.gov/issues/3907

Changes:
- create template.html from a catalog 404 page
- simplify the template to trim off unnecessary assets
- add static assets (img, css, fonts)
- change 500.html to use the new look.

### CODE REVIEW

**Comment by FuhuXia** [comment]
deployed to development
https://catalog-dev.data.gov/500.html

File: proxy/public/static-assets/css/d05bf0e7_fontawesome.css [hunk 1]
```diff
@@ -0,0 +1,7003 @@
...
+  font-family: "Font Awesome 5 Free";
+  font-display: block;
+  font-weight: 900;
+  src: url("/static-assets/fonts/fa-solid-900.woff2") format("woff2"), url("/static-assets/fonts/fa-solid-900.ttf") format("truetype"); }
+
+@font-face {
+  font-family: "Font Awesome 5 Free";
+  font-display: block;
+  font-weight: 400;
+  src: url("../../base/vendor/fontawesome-free/webfonts/fa-regular-400.woff2") format("woff2"), url("../../base/vendor/fontawesome-free/webfonts/fa-regular-400.ttf") format("truetype"); }
```
Comment (btylerburton) [inline proxy/public/static-assets/css/d05bf0e7_fontawesome.css:6982]: there's quite a few of these relative path links in the CSS and JS. How does the page look when it's rendered?

Comment (FuhuXia) [reply proxy/public/static-assets/css/d05bf0e7_fontawesome.css:6982]: these css relative links are not used. I only updated the minimal, actually being used, css links.

Comment (btylerburton) [reply proxy/public/static-assets/css/d05bf0e7_fontawesome.css:6982]: As long as the page looks good. There's a bunch of relative links in the JS file as well, but those are more than likely to unused modules.

**Review by btylerburton — APPROVED** [review]
⚡

File: proxy/public/static-assets/css/d05bf0e7_fontawesome.css (hunk 1 above)
Comment (FuhuXia) [reply proxy/public/static-assets/css/d05bf0e7_fontawesome.css:6982]: Yep. The way I use to tell whether we are missing some assets is looking at the network tab in the inspector. I did not see any 404. All necessary assets are identified and saved a local copy.

image

**Pull request merged**